hanami_application/config/providers/event_repository.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+Hanami.application.register_provider :event_repository do |container|
+  start do
+    require 'ruby_event_store/rom'
+    repository =
+      RubyEventStore::ROM::EventRepository.new(
+        rom: container['persistence.rom'],
+        serializer: RubyEventStore::NULL
```

Comment:
Changing this to `JSON` fixes the issue in https://github.com/RailsEventStore/rails_event_store/pull/1297 (at least for me)

```suggestion
        serializer: JSON
```